### ISSUE TITLE
Docs as Code section - Content APIs

### ISSUE THREAD

**Issue by lief-erickson** [briandominick/codewriting#18]
> 3. All content can be single-sourced, meaning tech writers and engineers maintain one canonical
source of truth, no matter how many places that content needs to appear in the generated docs.

This touches on, albeit without saying as much, as Docs as a Service or Content APIs. Content can be subscribed to and pull in based on requests. Consider pulling content into a mobile app, for instance. Too early to reference this sort of pipeline?

**Comment by briandominick** [comment]
Yeah this is great stuff. I don't know enough about it, and I knew even less when I was brushing up against it here.

**Comment by briandominick** [comment]
Lief I'm definitely interested in resources on this, or, frankly, a direct contribution.

**Comment by lief-erickson** [comment]
A Google search for "content api" turns up several resources. I have not used any of these, but am familiar that such things exist.

[Contentful](https://www.contentful.com/developers/docs/concepts/apis/)
[Seabourne](http://www.seabourneinc.com/what-is-a-content-api/)
[Programmableweb](https://www.programmableweb.com/category/content/apis?category=20149) seems to point to a lot of different tools making use of content APIs
